src/Core/Prefab/GlobeView.js

```diff
@@ -193,7 +193,7 @@ function GlobeView(viewerDiv, coordCarto, options = {}) {
     const sun = new THREE.DirectionalLight();
     sun.position.set(-0.5, 0, 1);
     sun.updateMatrixWorld(true);
-    this.scene.add(sun);
+    wgs84TileLayer.object3d.add(sun);
```

Comment:
Yes, that's why I won't change it back to `this.scene.add`

```js
wgs84TileLayer.object3d.add(sun);
```
Means exactly what @autra said "add the sun to the same root object".
It doesn't mean: "add the sun to the wgs84 layer".